### PR TITLE
[Node Manager] Add host_dvr_mac field in NodeInfo

### PR DESCRIPTION
**Pull request by kevin-zhonghao**
Add host_dvr_mac field in NodeInfo

### CODE REVIEW

File: web/src/main/java/com/futurewei/alcor/web/entity/NodeInfo.java [hunk 1]
```diff
@@ -83,6 +86,16 @@ public NodeInfo(String nodeId, String nodeName, String ipAddress, String macAddr
         this.gRPCServerPort = 0;
     }
 
+    public NodeInfo(String id, String name, String localIp, String macAddress, String veth, int gRPCServerPort, String host_dvr_mac) {
```
Comment (xieus) [inline web/src/main/java/com/futurewei/alcor/web/entity/NodeInfo.java:89]: As this is the latest constructor with full parameters, it contains the full implementation while existing constructors refer to this new one.

File: web/src/main/java/com/futurewei/alcor/web/entity/NodeInfo.java [hunk 2]
```diff
@@ -45,6 +45,9 @@
     @JsonProperty("server_port")
     private int gRPCServerPort;
 
+    @JsonProperty("host_dvr_mac")
+    private String host_dvr_mac;
```
[the remark this answers is not in the document]
Comment (kevin-zhonghao) [reply web/src/main/java/com/futurewei/alcor/web/entity/NodeInfo.java:49]: Sure

File: web/src/main/java/com/futurewei/alcor/web/entity/NodeInfo.java (hunk 1 above)
Comment (kevin-zhonghao) [reply web/src/main/java/com/futurewei/alcor/web/entity/NodeInfo.java:89]: I have a fix about this part ~ but I am not sure if it is correct ~

**Review by xieus — APPROVED** [review]
@kevin-zhonghao Thanks for the quick turnaround to make the change. Let us merge this PR to unblock the L3 port manager development. 

Meanwhile, please use the time to enhance the node manager UTs with this new field.

File: web/src/main/java/com/futurewei/alcor/web/entity/NodeInfo.java (hunk 1 above)
Comment (xieus) [reply web/src/main/java/com/futurewei/alcor/web/entity/NodeInfo.java:89]: Talked offline and addressed this comment.

**Pull request merged**